app/server/lib/ACLFormula.ts

```diff
@@ -105,4 +105,4 @@ function getAttr(value: any, attrName: string, valueNode: ParsedAclFormula): any
 function _compileAndCombine(args: ParsedAclFormula[], combine: (values: any[]) => any): AclEvalFunc {
   const compiled = args.map(_compileNode);
   return (input: AclMatchInput) => combine(compiled.map(c => c(input)));
-}
+}
```

Comment:
(most of code base goes with a newline here)

Reply:
Alright, if that's fine to you, I changed the `.editorconfig` file to reflect that: https://github.com/gristlabs/grist-core/pull/639/files#diff-0947e2727d6bad8cd0ac4122f5314bb5b04e337393075bc4b5ef143b17fcbd5bR9